07-modeling.Rmd

```diff
@@ -224,11 +228,11 @@ m_electric_multi <- recs_des %>%
   svyglm(
     design = .,
     formula = DOLLAREL ~ (Region + Urbanicity + TOTSQFT_EN + ACUsed) ^
-      2 - 1,
+      2 - 1, 
```

Comment:
This `-1` took me a minute to know what you were doing.  I'm not sure I knew this was a thing!  Can we add a little text around this to explain it?  Any why we are using it here?